models/mrsegmentator/meta.json

```diff
@@ -0,0 +1,140 @@
+{
+    "id": "3b8e2f1d-4c5a-4b2a-8f3e-6a7d9e2c1b0d", 
+    "name": "mrsegmentator",
+    "title": "MRSegmentator",
+    "summary": {
+      "description": "MRSegmentator is an AI-based pipeline for the segmentation of 40 anatomical structures in MR images (with and without contrast).",
+      "inputs": [
+        {
+          "label": "Input Image",
+          "description": "The MR or CT scan of a patient (Thorax, Abdomen and Pelvis).",
+          "format": "DICOM",
+          "modality": "MRI|CT",
+          "bodypartexamined": "WHOLEBODY",
+          "slicethickness": "n/a",
+          "non-contrast": true,
+          "contrast": true
+        }
+      ],
+      "outputs": [
+        {
+          "type": "Segmentation",
+          "classes": [
+            "SPLEEN",
+            "RIGHT_KIDNEY",
+            "LEFT_KIDNEY",
+            "GALLBLADDER",
+            "LIVER",
+            "STOMACH",
+            "PANCREAS",
+            "RIGHT_ADRENAL_GLAND",
+            "LEFT_ADRENAL_GLAND",
+            "LEFT_LUNG",
+            "RIGHT_LUNG",
+            "HEART",
+            "AORTA",
+            "INFERIOR_VENA_CAVA",
+            "PORTAL_AND_SPLENIC_VEIN",
+            "LEFT_ILIAC_ARTERY",
+            "RIGHT_ILIAC_ARTERY",
+            "ESOPHAGUS",
+            "SMALL_INTESTINE",
+            "DUODENUM",
+            "COLON",
+            "URINARY_BLADDER",
+            "SPINE",
+            "SACRUM",
+            "LEFT_HIP",
+            "RIGHT_HIP",
+            "LEFT_FEMUR",
+            "RIGHT_FEMUR",
+            "LEFT_AUTOCHTHONOUS_BACK_MUSCLE",
+            "RIGHT_AUTOCHTHONOUS_BACK_MUSCLE",
+            "LEFT_ILIOPSOAS",
+            "RIGHT_ILIOPSOAS",
+            "LEFT_GLUTEUS_MAXIMUS",
+            "RIGHT_GLUTEUS_MAXIMUS",
+            "LEFT_GLUTEUS_MEDIUS",
+            "RIGHT_GLUTEUS_MEDIUS",
+            "LEFT_GLUTEUS_MINIMUS",
+            "RIGHT_GLUTEUS_MINIMUS"
+          ]
+        }
+      ],
+      "model": {
+        "architecture": "U-net",
+        "training": "supervised",
+        "cmpapproach": "ensemble"
+      },
+      "data": {
+        "training": {
+          "vol_samples": 2649
+        },
+        "evaluation": {
+          "vol_samples": 960
+        },
+        "public": true,
+        "external": true
+      }
+    },
+    "details": {
+      "name": "MRSegmentator",
+      "version": "1.1.0",
+      "devteam": "AI-Assisted Healthcare Lab at Technical University Munich and Charité Universitätsmedizin Berlin",
+      "type": "nnU-Net (U-Net structure, optimized by data-driven heuristics)",
+      "date": {
+        "weights": "05/18/24",
+        "code": "n/a",
```

Comment:
The first code was released on May 12th, the version 1.1.0 was released on May 18th. I think it might be best to have both weights and code be the same date.